R/calculate_branching_point_feature_importance.R

```diff
@@ -58,7 +58,8 @@ calculate_branching_point_feature_importance <- function(
             milestone_id = factor(milestone_oi, levels = trajectory$milestone_ids),
             .data$feature_id,
             .data$importance
-          )
+          ) %>%
```

Comment:
Could also remove the `transmute()` entirely, instead using a `mutate()` for clarity, like:

```r
%>%
  mutate(milestone_id = factor(milestone_oi, levels = trajectory$milestone_ids)) %>%
  select(milestone_id, feature_id, importance)
```

Reply:
Thanks. I've updated the pull request. except using `all_of()` instead of nse.